nvidia-open-gpu-kernel-modules/lts/pkg.yaml

```diff
@@ -26,6 +26,8 @@ steps:
         cd kernel-open
 
         patch -p1 </pkg/patches/nvtophys.patch
+        patch -p2 </pkg/patches/psc.patch || (cat nvidia-uvm/uvm_hmm.c.rej && exit 1)
```

Comment:
Do we need this rejection test permanently in place? Or maybe some generic type of this should even be added to all the files to ensure no patch rejections happen

Reply:
this is my debug, I will clean it up